gravity.sh

```diff
@@ -160,48 +159,35 @@ function gravity_Schwarzchild() {
 
 	# Find all active domains and compile them into one file and remove CRs
 	echo "** Aggregating list of domains..."
-	truncate -s 0 $piholeDir/$matter
+	truncate -s 0 $piholeDir/$matterandlight
 	for i in "${activeDomains[@]}"
 	do
-   		cat $i |tr -d '\r' >> $piholeDir/$matter
+   		cat $i |tr -d '\r' >> $piholeDir/$matterandlight
 	done
 }
 
-# Pulsar - White/blacklist application
-function gravity_pulsar() {
 
+function gravity_Blacklist(){
 	# Append blacklist entries if they exist
-	if [[ -r $blacklist ]];then
-        numberOf=$(cat $blacklist | sed '/^\s*$/d' | wc -l)
-        echo "** Blacklisting $numberOf domain(s)..."
-        cat $blacklist >> $piholeDir/$matter
-	fi
+	blacklist.sh -f -nr -q
```

Comment:
Yeah, basically all that line says is "Run the blacklist script, force it to update the hosts file, don't reload dnsmasq, and do it quietly.